dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/ValidationResult.java

```diff
@@ -27,14 +27,32 @@
  */
 package org.hisp.dhis.tracker.validation;
 
+import java.util.List;
 import java.util.Set;
 
+import org.hisp.dhis.tracker.domain.Enrollment;
+import org.hisp.dhis.tracker.domain.Event;
+import org.hisp.dhis.tracker.domain.Relationship;
+import org.hisp.dhis.tracker.domain.TrackedEntity;
+
 /**
- * The result of the validation step. It contains a set of {@link Validation}s
- * divided in errors and warnings.
+ * ValidationResult is the result of the validation step. The tracked entities,
+ * enrollments, events and relationships can be persisted (created, updated,
+ * deleted) and thus trusted to be in a valid state. Any entity that passed
```

Comment:
Even if we risk to be repetitive I would specify here that we call these entities, `persistable entities`.
Then in the last sentence, Persistable entities it is cristal clear what we are talking about.

Reply:
done 😄